packages/wallets/torus/package.json

```diff
@@ -31,21 +31,24 @@
     },
     "dependencies": {
         "@solana/wallet-adapter-base": "^0.9.4",
-        "@toruslabs/solana-embed": "^0.1.5",
+        "@toruslabs/solana-embed": "^0.2.2",
```

Comment:
Does this version update have any breaking API changes we should be concerned with?

Reply:
No. There is no breaking API changes.
The minor version update is about the changes were in controllers only and doesn't affect the APIs used wallet adapter